types.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	imageutils "github.com/kyverno/kyverno/pkg/utils/image"
+)
+
+type ImageInfo struct {
+	imageutils.ImageInfo
+
+	// Pointer is the path to the image object in the resource
+	Pointer string `json:"jsonPointer"`
```

Comment:
How is this pointer built? At Kyverno side?

Reply:
Yes, this pointer is a part of `images` variable and is provided from kyverno

Although we should note that the current version of the [`apiutils` library](https://pkg.go.dev/github.com/kyverno/kyverno/pkg/utils/api#ImageInfo) doesn't expose it
```go
type ImageInfo struct {
    imageutils.ImageInfo
    // Pointer is the path to the image object in the resource
    Pointer string `json:"-"`
}
```